angular/projects/spark-angular/src/lib/components/sprk-dropdown/sprk-dropdown.component.spec.ts

```diff
@@ -32,25 +76,31 @@ describe('SprkDropdownComponent', () => {
     fixture = TestBed.createComponent(TestWrapperComponent);
     fixture.detectChanges();
     wrapperComponent = fixture.componentInstance;
-    dropDownComponent = fixture.debugElement.query(
+
+    dropdownComponent = fixture.debugElement.query(
       By.directive(SprkDropdownComponent),
     ).componentInstance;
-
     dropdownTriggerElement = fixture.nativeElement.querySelector('a');
+    dropdownIconElement = fixture.nativeElement.querySelector('svg');
     dropdownTriggerTextElement = fixture.nativeElement.querySelector('span');
     dropdownElement = fixture.nativeElement.querySelector('div');
+
+    dropdownComponent = fixture.debugElement.query(
+      By.directive(SprkDropdownComponent),
+    ).componentInstance;
+    dropdownTriggerElement = fixture.nativeElement.querySelector('a');
   });
 
   it('should create', () => {
-    expect(dropDownComponent).toBeTruthy();
+    expect(dropdownComponent).toBeTruthy();
   });
 
```

Comment:
Odd that we missed having a test for adding the base class. Let's add that with a length check to make sure that nothing additional is being adding.